roles/master/tasks/main.yml

```diff
@@ -1,14 +1,27 @@
 - name: initialize kubeadm on master
-  command: kubeadm init --pod-network-cidr=10.244.0.0/16 --token={{ kubernetes_token }} --apiserver-advertise-address={{ ansible_default_ipv4.address }}
+  shell: |
```

Comment:
Any particular reason for switching from command module to shell?  The former is generally preferred unless one needs to use shell redirection, pipes, etc.

Reply:
I couldn't break command into multiple lines, with either of these forms:

```
 command: >-
   long-multi
   line-command
```


or,


```
   command: |
    long-multi
    line command
```

Reply:
Make sense, and thanks for breaking into multiple lines!